packages/reactstrap-validation-date/src/AvDate.js

```diff
@@ -48,6 +48,12 @@ class AvDate extends Component {
     }
 
     this.state.focused = false;
+
+    const { getDefaultValue } = context.FormCtrl;
+
+    if (getDefaultValue(props.name)) {
```

Comment:
should this explicitly check for `undefined` ? What if someone specifies the default value for an input as `false` ?

Reply:
I was more so just following what the `AvDateRange` did with the default value. I think since this is for a `Date` field the user should be entering a `string` value regardless so we don't have to make that check.